src/pages/T4.js

```diff
@@ -0,0 +1,60 @@
+const { css } = require('emotion')
+const { theme } = require('../styles.js')
+const { html } = require('../utils.js')
+const Layout = require('../components/Layout.js')
+const ErrorList = require('../components/ErrorList.js')
+const LogoutLink = require('../components/LogoutLink.js')
+const SummaryTable = require('../components/SummaryTable.js')
+const ButtonLink = require('../components/ButtonLink.js')
+const ValidationError = require('../components/forms/ValidationError.js')
+const Checkbox = require('../components/forms/Checkbox.js')
+const Button = require('../components/forms/Button.js')
+
+const t4CSS = css`
+  position: relative;
+
+  > div {
+    margin-bottom: ${theme.space.xl};
+  }
+`
+
+const changesButtonCSS = css`
+  width: 200px;
+  background-color: pink;
+`
+
+const consentButtonCSS = css`
+  width: 200px;
+  float: right;
+`
+
+//todo How to get t4 info? Where does it get injected from? user object passed with an array of t4s?
+//is this passed in to the Dashbord object which then passes it to summary table?
```

Comment:
All good questions.

Currently it's just a user object that goes into the session.